### PR TITLE
UBERF-7446 & UBERF-7447: i18n fixes

### PR DESCRIPTION
**Pull request by ThetaDR**
Update the placeholder when language is changed.
Fallback to English if used language doesn't have needed string

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjgzMWE3MDc0ZmRjOGExYzc1OGFjZjkiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.tAr7wMtvYno94VyPRV483UdDl4kjxXpISTdpsZN0FDU">Huly&reg;: <b>UBERF-7477</b></a></sub>

### CODE REVIEW

File: packages/text-editor/src/components/TextEditor.svelte [hunk 1]
```diff
@@ -55,6 +55,15 @@
   let placeHolderStr: string = ''
 
   $: ph = translate(placeholder, {}, $themeStore.language).then((r) => {
+    if (editor !== undefined && placeHolderStr !== r) {
```
Comment (haiodo) [inline packages/text-editor/src/components/TextEditor.svelte:58]: better to start translation after editor is defined

Comment (aonnikov) [reply packages/text-editor/src/components/TextEditor.svelte:58]: editor must be defined with placeholder plugin configured with translated string. these changes is a dirty hack that updates plugin existing state when translation changes

Comment (ThetaDR) [reply packages/text-editor/src/components/TextEditor.svelte:58]: Nope, we need to translate even before the editor is defined – placeholder is set when the editor is defined, so it's required there

**Review by aonnikov — APPROVED** [review]

**Comment by haiodo** [comment]
@ThetaDR did failed test related?

**Comment by ThetaDR** [comment]
@haiodo shouldn't be related. Tests are failing due to some issues with the panel which I didn't change, so it's probably just unstable

**Review by haiodo — APPROVED** [review]

**Pull request merged**